lib/util/fields/increment_field.dart

```diff
@@ -31,48 +32,52 @@ class _IncrementFieldState extends State<IncrementField> {
 
   @override
   Widget build(BuildContext context) {
-    return Row(
-      children: <Widget>[
-        Expanded(
-          child: Align(
-            alignment: Alignment.centerLeft,
-            child: IconButton(
-              icon: const Icon(Icons.remove_circle_outline),
-              onPressed: _value <= widget.minValue
-                  ? null
-                  : () {
-                      _value -= 1;
-                      widget.onChanged(_value);
-                    },
+    return Card(
```

Comment:
Maybe not a card, but a border like for the other Inputs? and for people who don't get that that should be the seats write seats somewhere (Merri did not get it right away)

Reply:
Hm everybody in the user study got that part right away. Should probably wait for some more feedback there?